server/app/controllers/receive.py

```diff
@@ -57,7 +58,19 @@ class RequestArguments(TypedDict):
     'tca': fields.DateTime(required=True),
     'los': fields.DateTime(required=True),
     'sat': fields.Str(required=True),
-    'notes': fields.Str(required=False)
+    'notes': fields.Str(required=False),
+    'tle': fields.List(fields.Str, required=False,
+        validate=[
+            # TLE must contains exact two lines
+            validate.Length(equal=2, error="Require 2 line TLE. Actual: {input}"),
```

Comment:
Contrary to its name, TLE is often specified as 3 lines, with the first one being a name of the sat. That name is actually important. I've seen more than one solution where the name is used as a key. I strongly suggest to permit 2 and 3 lines here.

Reply:
Related to our talk:

1. We have satellite name in satellite table. It will be redundant data
2. Additional line doesn't have any official specification.
3. If we will want to use TLE we must to check if it is two or three line version. It complicated future solutions, but doesn't add any adventage.